tests/unit/TransactionalFolioNumber.spec.ts

```diff
@@ -0,0 +1,21 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { mount } from '@vue/test-utils'
+import { getVuexStore } from '@/store'
+import TransactionalFolionNumber from '@/components/common/TransactionalFolioNumber.vue'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+describe('Transactional Folio Number component', () => {
+  it('Displays expected content', () => {
+    const wrapper = mount(TransactionalFolionNumber, { vuetify, store })
+
+    // *** TODO: finish implementing tests
+    expect(wrapper.find('#transactional-folio-number-container').exists()).toBe(true)
+
+    wrapper.destroy()
+  })
+})
```

Comment:
You should try the new unit test format :) A lot of this set up is encapsulated in the `jest-wrapper-factory`!

Reply:
I'll look into that :)

Reply:
I left myself a comment (see next commit) to use the new unit test format.